src/main/java/org/sonar/squidbridge/annotations/RuleTemplate.java

```diff
@@ -24,6 +24,16 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
+ * Rule template should be manually registered as being template rule, when defining repositories.
+ *
+ * In order to register a rule as being rule template, the following method from SQ 6.7 LTS API should be used:
+ * <ul>
+ * <li>org.sonar.api.server.rule.RulesDefinition.NewRule.setTemplate(boolean)</li>
+ * </ul>
+ */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface RuleTemplate {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlpYaf12pdu-6jE9&open=AWHXxlpYaf12pdu-6jE9)

[ISSUE_KEY]: AWHXxlpYaf12pdu-6jE9